tests/test_pruner.cpp

```diff
@@ -229,11 +231,13 @@ template <class FT> int test_unpruned()
   pru.enumeration_radius = .85;
   double cost            = pru.single_enum_cost(pr);
   cerr << "Cost per enum " << cost << endl;
-
-  status       = (abs(1 - cost / 3.20e+10) > .02);
+  status |= (abs(1 - cost / 3.20e+10) > .02);
+  status |= std::isnan(cost);
+  cerr << std::isnan(cost) << endl;
```

Comment:
If you run it through `make check` but not if you run it manually. I'd prefer outputs which the user can understand, i.e. which helps to resolve potential bugs. Or is that unreasonable to expect here?